projects/js-packages/licensing/changelog/fix-user-licensing-error-message-icon

```diff
@@ -0,0 +1,4 @@
+Significance: minor
```

Comment:
Since that's "only" a CSS fix, I think this could be a "patch" change instead of a minor change. What do you think?

Note: updating this will also mean that the version numbers in all those files you modified will have to change.

Reply:
That was a mistake on my part – this is definitely a patch. Thanks for pointing it out.